src/Styleguide/Elements/__stories__/Typography.story.tsx

```diff
@@ -0,0 +1,85 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { Theme } from "../../theme"
+import { withInfo } from "@storybook/addon-info"
+import { Display, Sans, Serif } from "../Typography"
+
+const stories = storiesOf("Styleguide/Typography", module)
+
+stories
+  .add(
+    "Sans",
+    withInfo(`
+
+    https://www.notion.so/artsy/Typography-d1f9f6731f3d47c78003d6d016c30221
+
+  `)(() => {
+      return (
+        <Theme>
+          <div>
+            <Sans size="1">Donald Judd</Sans>
+            <Sans size="2">Donald Judd</Sans>
+            <Sans size="3">Donald Judd</Sans>
+            <Sans size="3t">Donald Judd</Sans>
+            <Sans size="4">Donald Judd</Sans>
+            <Sans size="4t">Donald Judd</Sans>
+            <Sans size="5">Donald Judd</Sans>
+            <Sans size="5t">Donald Judd</Sans>
+            <Sans size="6">Donald Judd</Sans>
+            <Sans size="8">Donald Judd</Sans>
+            <Sans size="10">Donald Judd</Sans>
+            <Sans size="12">Donald Judd</Sans>
+            <Sans size="14">Donald Judd</Sans>
+            <Sans size="16">Donald Judd</Sans>
```

Comment:
@alloy, you'd asked about size in the other PR. I sat down w/ Brian and asked him about what semantics would make the most sense for this component. That's what recommended. Still, it is a valid point that the Typography doc uses `modifier` instead of `size`.

Reply:
I'll ask in the design systems chat.